tests/testthat/test-ScpModel-Class.R

```diff
@@ -475,15 +482,16 @@ test_that("scpModelN", {
     se <- SummarizedExperiment()
     model <- ScpModel()
     metadata(se)[["test1"]] <- model
-    ## no model = error
+    ## Test changed :
```

Comment:
Very good, but no need to comment this. In a few commits, we will forget `n` was slot and your comment will become obsolete. Better remove it now. 

Aside that, I think 
```r
## no model = error
```
is a useful comment that will help debugging, so I would prefer to keep it.

Reply:
I just don't understand why we have this comment (## no model = error) here since the test is executed on a se with a ScpModel object.

Reply:
Agree this is not clear, something more informative would be something like: no estimated model = error